include/img/img-xbm.h

```diff
@@ -4,19 +4,8 @@
 
 struct lab_data_buffer;
 
-/**
```

Comment:
It would be good to keep comment documenting the function in `img-xbm.h`

Reply:
Alright. I once thought it could be removed as it's duplicated in `img.h`, but I agree that having a comment is better than none.